src/main/java/no/dcat/datastore/domain/dcat/builders/DcatBuilder.java

```diff
@@ -631,29 +629,31 @@ public void addSubjectContent(Subject subject, Resource resource) {
         }
     }
 
-    public void addDataDistributionService(DataDistributionService distributionService, Resource resource) {
-        if (distributionService != null) {
-            try {
-                Resource distributionServiceRes = null;
-                if (distributionService.getUri() != null) {
-                    distributionServiceRes = model.createResource(distributionService.getUri());
-                } else {
-                    distributionServiceRes = model.createResource(UUID.randomUUID().toString());
-                }
+    public void addDataDistributionService(List<DataDistributionService> distributionServices, Resource resource) {
+        if (distributionServices != null) {
+            distributionServices.forEach(distributionService -> {
+                try {
+                    Resource distributionServiceRes = null;
+                    if (distributionService.getUri() != null) {
+                        distributionServiceRes = model.createResource(distributionService.getUri());
+                    } else {
+                        distributionServiceRes = model.createResource(UUID.randomUUID().toString());
+                    }
 
-                distributionServiceRes.addProperty(RDF.type, DCATapi.DataDistributionService);
+                    distributionServiceRes.addProperty(RDF.type, DCATapi.DataDistributionService);
```

Comment:
Here we should also add a list of dcat:endpointURLs since it is one of the mandatory properties of a dcat:DataService, ref https://doc.difi.no/review/dcat-ap-no/#_obligatoriske_egenskaper_for_datatjeneste
Also we should add a list of dct:MediaTypes, since this is (probably) going to be a mandatory property as well: https://doc.difi.no/review/dcat-ap-no/#distribusjon-medietype

Reply:
Har sett litt på kode rundt dette og ser at det er behov for en større gjennomgang. Vi må definitivt komme tilbake til dette, så jeg setter inn en TODO enn så lenge.